integration/eclipse/package.json

```diff
@@ -1,42 +1,36 @@
 {
   "private": true,
-  "name": "eclipse-parent",
-  "version": "9.4.0-s44",
-  "engines": {
-    "yarn": "1.0.x || >=1.2.1",
-    "node": ">=16.15.0 <17"
-  },
-  "scripts": {
-    "prepare": "lerna run prepare",
-    "watch": "lerna run --parallel watch",
-    "build": "yarn install --ignore-scripts && lerna run build",
-    "lint": "lerna run lint --",
-    "test": "lerna run test",
-    "test:ci": "lerna run test:ci",
-    "publish": "yarn && yarn publish:latest",
-    "publish:next": "lerna publish --exact --canary=next --npm-tag=next --yes",
-    "update:next": "yarn upgrade -p \"sprotty|@eclipse-glsp/.*|@eclipse-glsp-examples\" --next "
+  "name": "@axonivy/eclipse-integration",
+  "version": "11.2.0-next",
+  "description": "Eclipse IDE integration webview for the Axon Ivy (GLSP-based) process editor",
+  "license": "(EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)",
+  "author": "Axon Ivy AG",
+  "dependencies": {
+    "@eclipse-glsp/client": "1.1.0-RC10",
+    "@eclipse-glsp/ide": "1.1.0-RC10",
+    "@eclipse-glsp/protocol": "1.1.0-RC10",
+    "@axonivy/process-editor": "^11.2.0-next"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.45.0",
-    "@typescript-eslint/parser": "^5.45.0",
-    "eslint": "^8.29.0",
-    "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-no-null": "^1.0.2",
-    "ignore-styles": "^5.0.1",
-    "lerna": "^6.1.0",
+    "@vscode/codicons": "^0.0.25",
```

Comment:
not sure if this is used anywhere anylonger...

Reply:
the integrations (eclipse-gslp/client) still need that